coffee-backend/src/main/kotlin/chat/letscoffee/friends/FriendRequestModel.kt

```diff
@@ -0,0 +1,23 @@
+package chat.letscoffee.friends
+
+import chat.letscoffee.security.model.User
+import javax.persistence.*
+
+
+@Entity
+@Table(name = "friendrequest")
+class FriendRequestModel(
+        @Id
+        @GeneratedValue(strategy = GenerationType.IDENTITY)
+        val id: Long? = null,
+        @ManyToOne
+        val from: User,
+        @ManyToOne
+        val to: User,
+        @Column
+        var accepted: Boolean? = null) {
```

Comment:
I suggest setting this as default false, making the type `Boolean` in the process, so that we don't have to deal with null values

Reply:
Or create another table `friends`, where we add a friend entry and remove a request entry in this table (Within a JDBC Transaction template) when accepting a request, so that we have a separate table of friends (and possibly more data later, or more tables referencing the `friends` table). That way we don't have to filter with by `accepted=true` when finding friends for a user

Reply:
The plan is to have a separate way of finding friends other than looking at FriendRequest. The reason for Boolean with null is that a request can have 3 values. 

Sent/Accepted/Rejected. 

It might be worth taking care of all of these values, (for example you can't delete a Rejected friend request)

Reply:
Good point, `null` makes sense 😊